Source/WebCore/css/StyleProperties.cpp

```diff
@@ -502,84 +497,39 @@ String StyleProperties::offsetValue() const
     ASSERT(offsetShorthand().length() == 5);
     StringBuilder result;
 
-    auto offsetPositionIndex = findPropertyIndex(CSSPropertyOffsetPosition);
-    auto offsetPathIndex = findPropertyIndex(CSSPropertyOffsetPath);
+    auto offsetPosition = propertyAt(findPropertyIndex(CSSPropertyOffsetPosition));
+    if (!offsetPosition.isImplicit())
+        result.append(offsetPosition.value()->cssText());
 
-    // Either offset-position and offset-path must be specified.
-    if (offsetPositionIndex == -1 && offsetPathIndex == -1)
-        return String();
-
-    if (offsetPositionIndex != -1) {
-        auto offsetPosition = propertyAt(offsetPositionIndex);
-        if (!offsetPosition.isImplicit()) {
-            if (!offsetPosition.value())
```

Comment:
I'm assuming it couldn't really happen that a property which isn't missing had a null value?

Reply:
That's right. No such thing.